First version of cyTOF visualization.

votti opened the pull request:
This is the our cyTOF visualization script.
It is based on the  20170920_example_analysis_v2.Rmd from the bbRexamples.

Please have a look if everything makes sense!

Also if somebody feels inspired to update things or add more comments, please feel free ;)

Commit:
First version of cyTOF visualization.

This is the first draft version of the cyTOF visualization script.
It is based on the  20170920_example_analysis_v2.Rmd from the bbRexamples.

toobiwankenobi commented on snippets/cytof/visualization/example_flowanalysis.Rmd:
```diff
@@ -0,0 +1,664 @@
+load_flowsom = F
+
+# filename of the tsne saved by the script
+# choose the filename of the tsne such that it reflects the tsne setting
```
`fn_tsne = file.path(out_folder, '20170222_tsne_out.rdata')`
`fn_phenograph = file.path(out_folder, '20170222_pheno_out.rdata')`
`fn_flowsom = file.path(out_folder, '20170222_flowsom_out.rdata')`

doesn't work for me to save objects with the path specified above (creates '//' file path separator)

could be changed to 

`fn_tsne = file.path(out_folder, '20170222_tsne_out.rdata', fsep='')`
`fn_phenograph = file.path(out_folder, '20170222_pheno_out.rdata', fsep='')`
`fn_flowsom = file.path(out_folder, '20170222_flowsom_out.rdata', fsep='')`

votti replied:
As an alternative fix, I removed trailing / at all folders. Can you check if this fixed the issue?

votti replied:
@toobiwankenobi Did this fix the issue?

toobiwankenobi replied:
@votti yep, this solved the issue.. sorry for my late reply.

toobiwankenobi reviewed (changes requested):
`file.path()` doesn't create valid paths.

Commit:
Removes trailing / of foldernames

This fixes the invalid file path issues created on some OS
if using file.path to create filepaths.

Commit:
Fixes merging of manual annotation

- The clusters should be factors
- If not all clusters are annotated, the clusters should be NA and not be removed.

Commit:
Updated example annotation to annotate all clusters

Commit:
Updates html after recent fixes.
